### ISSUE TITLE
Freeze while dragging a checkbox list

### ISSUE THREAD

**Issue by uli-on** [martinstoeckli/SilentNotes#107]
In hopes this might match the reason for the #105 issue, or at least shed some more light.

Possible precondition: 
- Have a text type note at the _topmost_ position (probably vice versa if you sort your notes top down).

Now:
- Create a checkbox list with some items, 4 would be the value I've had. Do not try an existing checkbox list, please, for some reason this here almost always needs newly created lists. The grain of magic powder.
- Close the checkbox list note
  - (You should find it at the top of the list of notes, at least I've it like that. Is the sorting order prerequisite to reproduce? Didn't test.)
- Press and hold the note at the cross arrow icon
- Now start dragging the note down. The moment the first hovered note jumps past your note, the screen will freeze. Also, try different speeds for the movement, the faster the more likely nothing bad will happen. At least according to _my_ experience.

It's not a bad freeze, though, there's several ways to un-glue the item, like shortly pressing the power button, using the app switcher, everything that causes a screen redraw, obviously. Even letting the note go and trying a second drag works at times to free the checkbox list.

And I really found that newly created lists do stick more reliably, similar to my observation that once a checkbox list has caused a freeze _and_ crossed an item it's less likely it will freeze again anytime soon. Just that, *less* likely.

- SilentNotes 5.8.1
- Android 6.0.1 (with GPS 21.18.16), non-rooted
- Webview 92.0.4515.159
- Moto G4

**Comment by martinstoeckli** [comment]
Thanks for the description. I know about this flaw, it happens in the Android webview since I added support for drag&drop with the "sortablejs" library. Somehow they are not entirely compatible, I suspect the events are a bit different in the webview and in other browsers, nevertheless I decided to keep this functionallity, because sorting with drag&drop is very useful.
It is certainly a candidate for further investigation, I already thought about replacing the sorting system with something as used by the to-do lists, this would it make easier to move a note to the bottom/top.
